### ISSUE TITLE
Support for other countries instead of only BR and US

### ISSUE THREAD

**Issue by aj07mm** [raphamorim/calendario#5]
the current support is only for BR and US it should support other countries as well

**Comment by raphamorim** [comment]
Yeah, I thinking in the next country. Suggestions?

**Comment by ikroeber** [comment]
Supporting other countries in the Americas would be wonderful, but I would also suggest including support for some European countries. UK would be a great option.

**Comment by raphamorim** [comment]
Good Idea @ikroeber, what's the bests countries in Europe to add support in calendario.

**Comment by ikroeber** [comment]
Maybe Germany or British schedules. I could try to take a look at the german one.
